source/common/protobuf/utility.cc

```diff
@@ -508,38 +509,59 @@ std::string MessageUtil::CodeEnumToString(ProtobufUtil::error::Code code) {
 
 namespace {
 
+std::unique_ptr<Protobuf::Message> typeUrlToNewMessage(Protobuf::MessageFactory& message_factory,
+                                                       absl::string_view type_url) {
+  const absl::string_view type_name = TypeUtil::typeUrlToDescriptorFullName(type_url);
+  const Protobuf::Descriptor* descriptor =
+      Protobuf::DescriptorPool::generated_pool()->FindMessageTypeByName(
+          static_cast<std::string>(type_name));
```

Comment:
You an static cast `string_view` to `std::string`? Live ad learn..

Reply:
Yup, there's a supported explicit cast operator (I also just learned this). Not that this is performance-critical at all, but it's probably slightly better in this case than initializing a string variable in this scope and then passing it in, because of copy elision.

Reply:
you can just omit the 'static_cast` and write:
```
  std::string(type_name)
```
as the actual.